### PR TITLE
Upped spring-core version too 5.3.19

### PR DESCRIPTION
**Pull request by dricjo**
Upped spring version to 5.3.19

### CODE REVIEW

File: server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java [hunk 1]
```diff
@@ -100,11 +100,26 @@ public String[] getBeanDefinitionNames() {
     return wrappedContext.getBeanDefinitionNames();
   }
 
+  @Override
+  public <T> ObjectProvider<T> getBeanProvider(Class<T> aClass, boolean b) {
```
Comment (Boris-Tang) [inline server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java]: can we keep the variable name 'allowEagerInit' as in the interface? it is easier to understand the meaning of the boolean

File: server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java [hunk 2]
```diff
@@ -100,11 +100,26 @@ public String[] getBeanDefinitionNames() {
...
+    return wrappedContext.getBeanProvider(aClass, b);
+  }
+
+  @Override
+  public <T> ObjectProvider<T> getBeanProvider(ResolvableType resolvableType, boolean b) {
```
Comment (Boris-Tang) [inline server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java]: same as above

File: server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java [hunk 3]
```diff
@@ -100,11 +100,26 @@ public String[] getBeanDefinitionNames() {
...
+    return wrappedContext.getBeanProvider(resolvableType, b);
+  }
+
   @Override
   public String[] getBeanNamesForType(ResolvableType resolvableType) {
     return wrappedContext.getBeanNamesForType(resolvableType);
   }
 
+  @Override
+  public String[] getBeanNamesForType(ResolvableType resolvableType, boolean b, boolean b1) {
```
Comment (Boris-Tang) [inline server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java]: same as above, b and b1 don’t make much sense

File: server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java [hunk 4]
```diff
@@ -140,6 +155,11 @@ public <A extends Annotation> A findAnnotationOnBean(String s, Class<A> aClass)
     return wrappedContext.findAnnotationOnBean(s, aClass);
   }
 
+  @Override
+  public <A extends Annotation> A findAnnotationOnBean(String s, Class<A> aClass, boolean b) throws NoSuchBeanDefinitionException {
```
Comment (Boris-Tang) [inline server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java]: 'b'

File: server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java [hunk 5]
```diff
@@ -205,6 +225,11 @@ public Class<?> getType(String s) throws NoSuchBeanDefinitionException {
     return wrappedContext.getType(s);
   }
 
+  @Override
+  public Class<?> getType(String s, boolean b) throws NoSuchBeanDefinitionException {
+    return null;
```
Comment (Boris-Tang) [inline server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java]: can we return something other than null?

Comment (dricjo) [reply server/src/main/java/se/sperber/cryson/spring/WrappingWebApplicationContext.java]: Yes certainly I missed the implementation on this one.

File: server/src/main/resources/cryson_database.xml [hunk 6]
```diff
@@ -25,13 +25,12 @@
     <property name="hibernateProperties">
       <value>
         hibernate.show_sql=false
-        hibernate.hbm2ddl.auto=update
-        hibernate.dialect=${cryson.database.dialect}
+        hibernate.hbm2ddl.auto=create
```
Comment (Boris-Tang) [inline server/src/main/resources/cryson_database.xml:28]: why we want to make this change?

File: server/src/test/java/se/sperber/cryson/apitest/CrysonAPITest.java [hunk 7]
```diff
@@ -266,7 +265,7 @@ public void shouldNotCommitInvalidEntities() throws Exception {
     postMethod.setRequestEntity(new StringRequestEntity(commitJson, "application/json", "UTF-8"));
     assertEquals(HttpStatus.SC_FORBIDDEN, httpClient.executeMethod(postMethod));
     assertEquals("{\"message\":\"CrysonTestEntity name size must be between 0 and 30\\n\",\"validationFailures\":[{\"entityClass\":\"CrysonTestEntity\",\"entityId\":" + entityId + ",\"keyPath\":\"name\",\"message\":\"size must be between 0 and 30\"}]}", postMethod.getResponseBodyAsString());
-  }
+  }*/
```
Comment (Boris-Tang) [inline server/src/test/java/se/sperber/cryson/apitest/CrysonAPITest.java]: Can we enable these two test cases?

**Review by Boris-Tang — APPROVED** [review]
LGTM, let's try

**Pull request merged**